core/include/detray/propagator/actor_chain.hpp

```diff
@@ -72,9 +75,8 @@ class actor_chain {
         (run(detail::get<indices>(_actors), states, p_state), ...);
     }
 
-    private:
```

Comment:
Do you need to remove this? You have `actors()` function already

Reply:
The ```private``` keyword is still there. But after the update it is not strictly needed anymore

Reply:
Oh there is one up there already...